packages/core/src/index.ts

```diff
@@ -252,10 +255,19 @@ export class Abby<
 
     const flagType = this._cfg.flags?.[key];
     const defaultValue = this._cfg.settings?.flags?.defaultValues?.[flagType!];
-
-    // return the defaultValue if exists
-    if (!storedValue && defaultValue != null) {
-      return defaultValue;
+    const fallbackValue = this._cfg.settings?.flags?.fallbackValues?.[key];
+
+    // if we dont have a stored value
+    // we first check if we have a specified fallback value for this flag
+    // otherwise we check if we have a specified default value for this flag type
+    // otherwise we return undefined
+    if (!storedValue) {
+      if (fallbackValue != null) {
```

Comment:
still not sure why these values arent allowed? we could easily use `key in obj`?